litestar/plugins.py

```diff
@@ -161,3 +159,41 @@ def to_openapi_schema(self, annotation: Any, schema_creator: SchemaCreator, dto_
     OpenAPISchemaPluginProtocol,
     CLIPluginProtocol,
 ]
+
+PluginT = TypeVar("PluginT", bound=PluginProtocol)
+
+
+class PluginRegistry:
+    __slots__ = {
+        "init": "Plugins that implement the InitPluginProtocol",
```

Comment:
I haven't seen `__slots__` as a dict, is this new?

Reply:
It's not new, but their usage for documentation has only been added in 3.9. Before that it was "left up for future implementation" to decide what the meaning of this was iirc.